jax/experimental/global_device_array.py

```diff
@@ -392,13 +407,19 @@ def from_callback(cls, global_shape: Shape, global_mesh: pxla.Mesh,
 
     ``data_callback`` is used to fetch the data for each local slice of the returned GlobalDeviceArray.
 
-    Example::
+    Example:
 
-      global_input_shape = (8, 2)
-      global_input_data = np.arange(prod(global_input_shape)).reshape(global_input_shape)
-      def cb(index):
-        return global_input_data[index]
-      gda = GlobalDeviceArray.from_callback(global_input_shape, global_mesh, mesh_axes, cb)
+      >>> from jax.experimental.maps import Mesh
+      >>> import numpy as np
+      >>> global_input_shape = (8, 8)
+      >>> mesh_axes = ['x', 'y']
+      >>> global_mesh = global_mesh = Mesh(np.array(jax.devices()).reshape(2, 4), ('x', 'y'))
+      >>> global_input_data = np.arange(prod(global_input_shape)).reshape(global_input_shape)
+      >>> def cb(index):
```

Comment:
Add line breaks before and after function definitions for readability (here and below)

Reply:
DOne